README.md

```diff
@@ -1085,6 +1088,217 @@ cardNumber.setAltText("Card Number");
 //clear altText
 cardNumber.clearAltText(); 
 ```
+
+## Using Skyflow File Element to upload a file
+
+Like other kinds of data, you can upload binary files in your vault using Skyflow File Element. Follow the steps below to securely upload a file.
+### Step 1: Create a container
+
+First create a container for the form elements using the `container(Skyflow.ContainerType)` method of the Skyflow client as show below:
+
+```javascript
+const container = skyflowClient.container(Skyflow.ContainerType.COLLECT)
+```
+
+### Step 2: Create a file Element
+
+A Skyflow collect Element is defined as shown below: 
+
+```javascript
+const collectElement =  {
+   table: "string",             //the table this data belongs to
+   column: "string",            //the column into which this data should be inserted
+   skyflowID: "string",         // the skyflow_id of the record
+   type: Skyflow.ElementType,   //Skyflow.ElementType enum
+   inputStyles: {},             //optional styles that should be applied to the form element
+   labelStyles: {},             //optional styles that will be applied to the label of the collect element
+   errorTextStyles:{},          //optional styles that will be applied to the errorText of the collect element
+}
+```
+The `table` and `column` fields indicate which table and column in the vault the Element corresponds to. 
+
+`skyflowID` indicates the records in which the file will be uploaded.
+
+**Note**: 
+- `skyflowID` is required while creating File element
+-  Use dot delimited strings to specify columns nested inside JSON fields (e.g. `address.street.line1`)
+
+ ## Step 3: Mount Elements to the DOM
+
+To specify where the Elements will be rendered on your page, create placeholder `<div>` elements with unique `id` tags. For instance, the form below has 4 empty divs with unique ids as placeholders for 4 Skyflow Elements. 
+
+```html
+<form>
+  <div id="file"/>
+  <br/>
+  <button type="submit">Submit</button>
+</form>
+```
+
+Now, when the `mount(domElement)` method of the Element is called, the Element will be inserted in the specified div. For instance, the call below will insert the Element into the div with the id "#cardNumber".  
```

Comment:
"Now, when the `mount(domElement)` method of the Element is called, the Element is inserted in the specified div. For instance, the call below inserts the Element into the div with the id "#cardNumber"."